libs/akita/package.json

```diff
@@ -28,7 +28,7 @@
   "license": "Apache-2.0",
   "author": "Netanel Basal",
   "dependencies": {
-    "schematics-utilities": "^1.1.1"
+    "schematics-utilities": "^2.0.2"
```

Comment:
This is a breaking change. Did you check that the schematics work as before?

Reply:
as mentioned in the PR, this is a bug because the library was compiled with `2.0.2` and published with `1.1.1`, fortunately the used API seems consistent because this could caused problems before.